pkgs/applications/video/bilibili/default.nix

```diff
@@ -41,9 +41,10 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Electron-based bilibili desktop client";
     homepage = "https://github.com/msojocs/bilibili-linux";
-    license = licenses.mit;
+    license = with licenses; [ unfree mit ];
     maintainers = with maintainers; [ jedsek kashw2 ];
-    platforms = platforms.unix;
+    platforms = [ "x86_64-linux" ];
```

Comment:
I'm pretty sure the package once worked fine on aarch64, so bad it doesn't now...

Reply:
https://github.com/NixOS/nixpkgs/blob/feeae486deb50aab190d1dd46626a6c5f9f3f698/pkgs/applications/video/bilibili/default.nix#L11-L14

We used compiled amd64 binaries， should download aarch64 version for aarch64-linux，but I don't have a machine to test